sys-boot/os-prober/os-prober-1.76-r1.ebuild

```diff
@@ -1,7 +1,7 @@
 # Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
```

Comment:
what is the purpose of EAPI bump?

Reply:
Main purpose of the PR was adding myself as a Proxy Maintainer and I thought I would bump the EAPI since it does not introduce any errors. However the rest of the Ebuild remains unchanged so it does not really make use of the changes included in EAPI 7.
Then, should I just leave it in EAPI 6 if I do not include anything in the Ebuild that requires the EAPI to be EAPI 7?

Reply:
You can't change ebuild's EAPI in place, especially in a stable ebuild. Please revbump it and then I'll review the new revision.